src/compile_torture_tests.py

```diff
@@ -81,9 +73,9 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config):
   result = testing.execute(
       tester=testing.Tester(
           command_ctor=c_compile,
-          outname_ctor=Outname(suffix),
```

Comment:
How does this pass `extras` to `create_outname`?
https://github.com/WebAssembly/waterfall/blob/master/src/testing.py#L78 only has it called with `outdir, infile`